### PR TITLE
[Merged by Bors] - feat(CategoryTheory/Monoidal): add lemmas for the whiskerings

### PR DESCRIPTION
**Pull request by yuma-mizuno**
Extracted from #6307. 

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Monoidal/Category.lean [hunk 1]
```diff
@@ -222,7 +222,79 @@ end MonoidalCategory
 open scoped MonoidalCategory
 open MonoidalCategory
 
-variable (C : Type u) [𝒞 : Category.{v} C] [MonoidalCategory C]
+variable {C : Type u} [𝒞 : Category.{v} C] [MonoidalCategory C]
+
+namespace MonoidalCategory
+
+@[reassoc (attr := simp)]
+theorem hom_inv_whiskerLeft (X : C) {Y Z : C} (f : Y ≅ Z) :
```
Comment (joelriou) [inline Mathlib/CategoryTheory/Monoidal/Category.lean:230]: I may suggest a change:
```suggestion
theorem whiskerLeft_hom_inv (X : C) {Y Z : C} (f : Y ≅ Z) :
```
and similarly for `inv_hom_whiskerLeft`. (But, the `hom_inv_whiskerRight` look good to me.)

**Review by joelriou — APPROVED** [review]

File: Mathlib/CategoryTheory/Monoidal/Category.lean (hunk 1 above)
Comment (yuma-mizuno) [reply Mathlib/CategoryTheory/Monoidal/Category.lean]: Thanks! I also renamed the similar lemmas for bicategories for consistency.

Comment (joelriou) [reply Mathlib/CategoryTheory/Monoidal/Category.lean]: Thanks! The same change should be done also for `inv_hom_whiskerLeft'` and `hom_inv_whiskerLeft'`.

**Comment by joelriou** [comment]
Thanks! 

bors merge

**Pull request closed without merging**